charmcraft.yaml

```diff
@@ -31,40 +31,43 @@ parts:
       just requirements
       craftctl default
 
-subordinate: true
+charm-libs:
+  - lib: operator-libs-linux.apt
+    version: "0.15"
+  - lib: operator-libs-linux.systemd
+    version: "1.4"
 
-peers:
-  storage-peers:
-    interface: storage_peers
+subordinate: true
 
 requires:
-  fs-share:
-    interface: fs_share
+  filesystem:
+    interface: filesystem_info
   juju-info:
     interface: juju-info
     scope: container
 
 config:
   options:
-    mountinfo:
+    mounts:
```

Comment:
This should probably be called mount-options or something similar

Reply:
For the local mountpoint, that's still needed - but it should be its own configuration value which takes a string value (Unix Path)